ethicalads-theme/templates/ea/homepage.html

```diff
@@ -111,7 +111,9 @@ <h3>Transparent</h3>
 
       <div class="col-md-6 mb-3">
         <h2 class="font-weight-bold mb-8">High-performing developer ads</h2>
-        <p class="font-size-lg text-gray-800">Reach developers on quality sites and connect with your exact tech audience.</p>
+        <p class="font-size-lg text-gray-800">Reach developers on quality sites and connect with your exact tech niche with powerful contextual targeting. Whether you are focused on data scientists or full stack, we ensure your ads are shown to the right audience.</p>
```

Comment:
This first line is a mouth full and has two different verbs (reach, connect). I'd probably break it up? We are also focusing again on the publishers pretty heavily here. We should do these in separate sections probably, otherwise the message gets muddled. Maybe we just want to focus on publishers here? Not 100% sure.  

```suggestion
        <p class="font-size-lg text-gray-800">Reach your exact developer niche with powerful contextual targeting. Whether you are focused on data scientists or full stack devs, we ensure your ads are shown to the right audience.</p>
```